framework/src/actions/CheckOutputAction.C

```diff
@@ -50,10 +50,10 @@ CheckOutputAction::checkVariableOutput(const std::string & task)
   {
     // Loop through the actions for the given task
     const std::vector<Action *> & actions = _awh.getActionsByName(task);
-    for (std::vector<Action *>::const_iterator it = actions.begin(); it != actions.end(); ++it)
+    for (auto act : actions)
```

Comment:
> On May 27, 2016, at 9:06 AM, Derek Gaston <notifications@github.com> wrote:
> 
> In framework/src/actions/CheckOutputAction.C:
> 
> > @@ -50,10 +50,10 @@ CheckOutputAction::checkVariableOutput(const std::string & task)
> >    {
> >      // Loop through the actions for the given task
> >      const std::vector<Action *> & actions = _awh.getActionsByName(task);
> > -    for (std::vector<Action *>::const_iterator it = actions.begin(); it != actions.end(); ++it)
> > +    for (auto act : actions)
> I would also argue that we still want to use const auto & here for consistency.
> 
> What do you guys think?
> 

I was unsure what to do for naked pointers, so didn't use &.  I don't think there could possibly be a performance penalty for doing it, so I can change it.